kaska/inference_runner.py

```diff
@@ -258,6 +264,10 @@ def kaska_runner(
         A = dask_client.map(wrapper, them_chunks)
         retval = dask_client.gather(A)
 
-    parameter_names = retval[0]
-
+    try:
+        parameter_names = next(item for item in reval if item is not None)
```

Comment:
I don't really understand why to take `next` if we are creating the iterator every time. I imagine it's different for each dask client, is that correct?

Reply:
All parts of `retval` (there's a typo there as well) should return either `None` or some information. The information should be the same for all the elements of `retval`, so we just want the first "not `None`" element, if that makes sense?